package.json

```diff
@@ -19,10 +19,11 @@
   "homepage": "https://github.com/behaviorbot/update-docs#readme",
   "devDependencies": {
     "eslint-config-probot": "^0.1.0",
```

Comment:
nit: `eslint-config-probot` is no longer used and should be removed from the `package.json`